app/static/js/index.js

```diff
@@ -54,6 +54,7 @@ schoolYCCheckbox.checked = true;
 
 p.clearFilters.onclick = function () {
 	resetFilters();
+	p.query.value = "";
```

Comment:
Super nitpicky: can you use single quotes to match the rest of the file?

Reply:
Copilot like double quotes LMAO

I also like double quotes because I come from C# where doublequote is string and singlequote is char. Will change tho. See you at hackathon today!

Reply:
If you want to convert the standard throughout the codebase you are welcome to (although I don't think that's a particularly great use of time) – but for now I would just prefer being consistent with the existing standard

Reply:
It looks like the rest of the file uses double quotes 👀